test/src/test/integration/librad/scenario/tracked_references.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright © 2019-2020 The Radicle Foundation <hello@radicle.foundation>
```

Comment:
This is the old license header, let's stick to the updated one for all new files.

Reply:
You're referring to this right?:

```
// Copyright © 2021 The Radicle Link Contributors
//
// This file is part of radicle-link, distributed under the GPLv3 with Radicle
// Linking Exception. For full terms see the included LICENSE file.
```

Reply:
yus